src/main/java/core/basesyntax/WorkWithFile/FileWriterImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.WorkWithFile;
+
+import core.basesyntax.WorkWithFileInterface.FileWriter;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String report, String fileName) {
+        try {
+            Files.write(Paths.get(fileName), report.getBytes());
+        } catch (IOException e) {
+            System.out.println("Error writing to file: " + fileName);
```

Comment:
Consider replacing `e.printStackTrace();` with a more informative logging mechanism or rethrowing the exception with a custom message. This will help in identifying the issue more effectively in a production environment.